latex-documents/practical_framework/src/Sprint-5.tex

```diff
@@ -0,0 +1,81 @@
+
+\section{Practical Framework}
```

Comment:
I think this section was already defined on sprint 0 file, did you export the team document to see the result?

Reply:
Yes and this is the result, that \section PracticalFramework its for Sprint 5 form part of that Section, like it is Sprint 0.
![Screenshot from 2024-06-27 11-31-00](https://github.com/Pending-Name-21/miscellany/assets/90282510/a3de3170-05c7-4ad8-874d-749d8186ff01)